pkg/controller/clusterdeployment/metrics.go

```diff
@@ -21,14 +21,14 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "hive_cluster_deployment_install_job_duration_seconds",
 			Help:    "Distribution of the runtime of completed install jobs.",
-			Buckets: []float64{60, 300, 600, 1200, 1800, 2400, 3000, 3600},
+			Buckets: []float64{1800, 2400, 3000, 3600, 4500, 5400, 7200},
```

Comment:
Same comments as below for this one @newtonheath - we need more buckets between 1800 and 3600, and maybe just a value below and/or above the lower and upper thresholds 
![image](https://user-images.githubusercontent.com/54181650/167930119-3f07a28f-613e-4e49-b8f8-495ce28bfa94.png)

Reply:
Also seen installer higher than 1 hour more than 30 days ago. Again, some more detail above that can help us how bad it get above the threshold as we investigate

![image](https://user-images.githubusercontent.com/91277/167942272-71a6d78c-922b-4e6c-a9a4-a2d978974ad5.png)

Reply:
Alright, let's push this in, provided we remove some buckets on a later date once we have a good idea of the threshold